qiskit_nature/problems/__init__.py

```diff
@@ -17,9 +17,25 @@
 .. currentmodule:: qiskit_nature.problems
 
 
+Second-Quantization Problems
+++++++++++++++++++++++++++++
+
 .. autosummary::
    :toctree:
 
    second_quantization
 
+Sampling Problems
+++++++++++++++++++++++++++++
```

Comment:
You may need to fix the heading line here.. If not, Sphinx might complain about the mismatching length w.r.t. the title.

Reply:
Update: same in more cases to come...

Reply:
Thanks. Did not know that lengths of both are checked against each other.